src/Service/NotificationService.php

```diff
@@ -0,0 +1,193 @@
+{
+    public function __construct(private string $emailFromAddress, private MailerInterface $mailer)
+    {
+    }
+
+    /**
+     * @param $booking
+     * @param $resource
+     * @param string $type
+     *
+     * @return void
+     */
+    public function sendBookingNotification($booking, $resource, string $type)
+    {
+        try {
+            $data = [
+                'booking' => $booking,
+                'resource' => $resource,
+                'user' => [
+                    'name' => $booking->getUserName(),
+                    'mail' => $booking->getUserMail(),
+                ],
+                'metaData' => $booking->getMetaData(),
+            ];
+
+            $notification = $this->buildNotification($type, $data);
+
+            $this->sendNotification($notification);
+        } catch (JsonException $e) {
+        }
+    }
+
+    /**
+     * @param string $type
+     * @param array $data
+     *
+     * @return array
+     */
+    private function buildNotification(string $type, array $data): array
+    {
+        $notificationData = [];
+        try {
+            $template = null;
+            $fileAttachments = [];
+            $to = $data['user']['mail'];
+            $subject = 'Booking bekræftigelse: '.$data['resource']->getResourceName().' - '.$data['resource']->getLocation();
+            switch ($type) {
+                case 'success':
+                    $template = 'emailBookingSuccess.html.twig';
+
+                    $events = $this->prepareIcalEvents($data);
+                    $iCalendarComponent = $this->createCalendarComponent($events);
+
+                    $fileAttachments = [
+                        'ics' => [$iCalendarComponent],
+                    ];
+                    break;
+                case 'booking_changed':
+                    $template = 'emailBookingChanged.html.twig';
+                    $subject = 'Booking ændret: '.$data['resource']->getResourceName().' - '.$data['resource']->getLocation();
+                    break;
+                case 'booking_failed':
+                    $template = 'emailBookingFailed.html.twig';
+                    $subject = 'Booking lykkedes ikke: '.$data['resource']->getResourceName().' - '.$data['resource']->getLocation();
+                    break;
+            }
+            $notificationData = [
+                'from' => $this->emailFromAddress,
+                'to' => $to,
+                'subject' => $subject,
+                'template' => $template,
+                'data' => $data,
+                'fileAttachments' => $fileAttachments,
+            ];
+        } catch (Exception $e) {
+        }
+
+        return $notificationData;
+    }
+
+    /**
+     * @param array $notification
+     *
+     * @return void
+     */
+    private function sendNotification(array $notification): void
+    {
+        try {
+            $email = (new TemplatedEmail())
+        ->from($notification['from'])
```

Comment:
I think we need to add this rule as well to php-cs-fixer:
https://cs.symfony.com/doc/rules/whitespace/method_chaining_indentation.html
like array_indentation
https://github.com/itk-dev/book_aarhus/blob/develop/.php-cs-fixer.dist.php#L10